onnxruntime/test/python/transformers/test_flash_attn_cuda.py

```diff
@@ -2082,6 +2087,8 @@ def gqa_no_past_memory_efficient_test_cases():
             for n, n2 in num_h:
                 for h in h_sizes:
                     for rotary, rotary_interleaved in [(True, False), (True, True), (False, False)]:
+                        if rotary and platform.system() == "Windows":
+                            continue
```

Comment:
nit: I think it is better to skip a test in the test body instead of skipping a test parameterization. pytest has `pytest.skip` or `@pytest.mark.skip`.